Contents/Code/__init__.py

```diff
@@ -603,25 +686,136 @@ def update(self, metadata, media, lang):
+                                                    else:
+                                                        metadata.rating_image = rating_info['image_bad']
+                                                
+                                                metadata.rating = add_rating_value
+                                                metadata.rating_count = add_votes
+
+                                                if audience_ratings_found == 0:
+                                                    log.debug("No Audience ratings found, setting them based on Critic Rating in case none provided")
+                                                    metadata.audience_rating = metadata.rating
+                                                    metadata.audience_rating_image = metadata.rating_image
+
+                                            elif rating_info['type'] == 'audience' and audience_default_found == False:
+                                                audience_ratings_found += 1
+                                                audience_score_total += add_rating_value
+                                                
+                                                if rating_default == True: # use default provider for rating
+                                                    critic_default_found = True
+                                                    log.debug("Audience Default rating set, will not average scores")
+                                                else: # use average score for rating
+                                                    add_rating_value = round(float(audience_score_total / audience_ratings_found), 1)
+                                                    log.debug("Average Audience Score: " + str(add_rating_value))
+                                                
+                                                # use image from default or rating with most votes
+                                                if (add_votes > audience_votes or rating_default == True) and rating_info['image_good'] and rating_info['image_bad']:
+                                                    if add_rating_value >= rating_info['score_good']:
+                                                        metadata.audience_rating_image = rating_info['image_good']
+                                                    else:
+                                                        metadata.audience_rating_image = rating_info['image_bad']
+                                                
+                                                metadata.audience_rating = add_rating_value
+                                                metadata.rating_count = add_votes #audience_rating_count doesn't exist
+                                                
+                                                if critic_ratings_found == 0:
```

Comment:
While the intention is good, I don't agree that we want to do this? In my testings, we just get the same rating shown twice. With the same logo.

Was the intention to have two audience ratings shows side by side? Or what was the idea here?